kmath-core/src/commonMain/kotlin/scientifik/kmath/expressions/Builders.kt

```diff
@@ -0,0 +1,23 @@
+package scientifik.kmath.expressions
```

Comment:
There is no class called `Builders`. I suggest starting the name with lower case letter.

Reply:
This is a violation of Coding Conventions. 
![image](https://user-images.githubusercontent.com/38042667/84574662-1d07a880-add2-11ea-82bb-fb1cacc5be07.png)

Reply:
Yes, but there are some variations of file naming accepted by the communitiy. And there is a style that is used accross other modules, so we need to keepr it uniform. I will write a doc file with local conventions.